### ISSUE TITLE
why there is no bet?

### ISSUE THREAD

**Issue by gftyzz12** [EmpireAssatoken/Pancakeswap-Predcition-Winnerbot#2]
hello sir! why there is no bet?

**Comment by EmpireAssatoken** [comment]
Hey, what kind of no bet?

**Comment by EmpireAssatoken** [comment]
Have u do that? 

Open the bot.js file and setup the following variables:
BET_AMOUNT: 5, // Amount of each bet (In USD)
DAILY_GOAL: 20, // Total profit you are aiming to earn (In USD)

Do u bet with BNB? U need to have bnb on your wallet. What system do u use?

**Comment by gftyzz12** [comment]
Hello, I have BNB in my account. I have been running the program for about a day, but I just checked it and there is no betting. I have modified bot.js according to the tutorial.

**Comment by gftyzz12** [comment]
I use ubuntu to run this program,it has always been like this ↓↓
> pancakeswap-prediction-smartbot@1.0.0 start /home/pgp/Pancakeswap-Predcition-Winnerbot
> node bot.js

🤗 Welcome! Waiting for next round... 
Your balance is enough: 0.323264139742528013 BNB

**Comment by EmpireAssatoken** [comment]
Yes, unfortunately this is a bug... we are working on it. try it with a second metamask wallet, because it may be because the bug comes from it. let me know if it worked.

**Comment by EmpireAssatoken** [comment]
try to send your amount to bnb to the new address and then copy and paste the private key into the .env. update best with apt update your system.

**Comment by gftyzz12** [comment]
ok I will try it now

**Comment by EmpireAssatoken** [comment]
Let me know if it works :)

**Comment by EmpireAssatoken** [comment]
if the whole thing doesn't work and you happen to have a second device, then try it with that. If you don't have a second device, try using a VM like Virtualbox and update the system well :)

**Comment by gftyzz12** [comment]
I changed the wallet and updated the system, does it still need to wait for the signal?

**Comment by gftyzz12** [comment]
Still the same, it no other run
> pancakeswap-prediction-smartbot@1.0.0 start /home/pgp/Pancakeswap-Predcition-Winnerbot
> node bot.js

🤗 Welcome! Waiting for next round... 
Your balance is enough: 0.323063522276349324 BNB

**Comment by EmpireAssatoken** [comment]
That's really weird. The bot works though.
VM already tried? Maybe he doesn't really like the system...

**Comment by gftyzz12** [comment]
ok thank you anyway!

**Comment by EmpireAssatoken** [comment]
I have the error

**Comment by EmpireAssatoken** [comment]
Try your best and let me please know if it works for our Team for the Fix :)

**Comment by gftyzz12** [comment]
THRESHOLD: 100 // Minimum % of certainty of signals (50 - 100 )I've set it up like this, it shouldn't bet immediately

**Comment by gftyzz12** [comment]
I ran it on my other server and the result was the same, I ran it overnight and I'll check back in the morning

**Comment by EmpireAssatoken** [comment]
Do u start with npm -I or npm start? 
that’s weird..

**Comment by gftyzz12** [comment]
npm start

**Comment by EmpireAssatoken** [comment]
I thank we have the issue

**Comment by EmpireAssatoken** [comment]
Our team update the source in 5-10 minutes

**Comment by EmpireAssatoken** [comment]
So, try please with git clone

**Comment by EmpireAssatoken** [comment]
If it doesn’t work please edit .env and edit follow to this: 

BSC_RPC=https://bsc-dataseed1.ninicoin.io/

**Comment by gftyzz12** [comment]
still like that
🤗 Welcome! Waiting for next round...
Your balance is enough: 0.323063522276349324 BNB
^C
(base) pgp@vmi1187281:~/Pancakeswap-Predcition-Winnerbot$ npm start

> pancakeswap-prediction-smartbot@1.0.0 start /home/pgp/Pancakeswap-Predcition-Winnerbot> node bot.js

🤗 Welcome! Waiting for next round...
Your balance is enough: 0.323063522276349324 BNB

**Comment by gftyzz12** [comment]
> 如果它不起作用，请编辑 .env 并按照以下内容进行编辑：
> 
> BSC_RPC= https://bsc-dataseed1.ninicoin.io/

i will try this

**Comment by EmpireAssatoken** [comment]
I think that’s not the problem…

**Comment by EmpireAssatoken** [comment]
u can also try this:  https://bsc-dataseed1.defibit.io/
